Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM registry.centos.org/centos/centos:7
   
```

Comment:
Same changes has to be repeated for Dockerfile.rhel and Dockerfile.new

Reply:
Ack